main.js

```diff
@@ -152,3 +152,23 @@ function changeBtnText() {
     showStarredBtn.innerText = "Show Starred Ideas";
   }
 }
+
```

Comment:
@butlertree Do you think we could find a way to separate the event listeners and make these named functions that we then call from the listeners instead of having anonymous functions inside the listeners themselves? You can take a look at what I did with the `toggleFaveButton` function, for an example of what I'm talking about. 

If it's too difficult to refactor, this totally works. But I think it would be worth a try to make the code cleaner and more scalable.

Reply:
I attempted to refactor.  With the limited time I was unable to make significant changes to the current code.  However as it is the code is functioning.

Reply:
@butlertree Agreed, it's totally working fine as is. Might be something to come back to for the sake of learning and making it more scalable later on, but I'm 100% a-ok with merging as is! Thanks for giving it a try! 🍪